### PR TITLE
ENH: Call `braintransform` script directly

### PR DESCRIPTION
**Pull request by jhlegarreta**
Call `braintransform` script directly.

### CODE REVIEW

**Comment by jhlegarreta** [comment]
Tested on your fork @Remi-Gau, this should now work:
https://github.com/Remi-Gau/global2021/commits/main

Successful Action run:
https://github.com/Remi-Gau/global2021/runs/4287821331?check_suite_focus=true#step:5:8

I presume the site built based on your fork has been updated successfully:
https://remi-gau.github.io/global2021/projects/

So what made PR #102 cause issues was to call the `transform_issues_to_pages.py` script using `python` in the actions workflow file, and then to fail at adding `requests` package to the requirements.

We may want to wait until Donostia's event is over before merging.

**Comment by Remi-Gau** [comment]
OK let's merge this tomorrow when BHD is over.

**Review by Remi-Gau — APPROVED** [review]

**Pull request merged**
